tensorflow/contrib/go/g3doc/graph.md

```diff
@@ -0,0 +1,235 @@
+# Graph
+
+```Go
+type Graph struct {
+    // contains filtered or unexported fields
+}
+```
+
+Graph Representation of the computation graph.
+
+## Graph Constructors
+
+### LoadGraphFromFile
+
+```go
+func LoadGraphFromFile(path string) (gr *Graph, err error)
```

Comment:
I think that even when the var names are not going to be used internally, I prefer to name the returned parameters because when you read the documentation generated by godoc sometimes this helps.
For this method, it is obvious what the returned vars will contain, but if you have something like:
```go
func Coordinates() (int, int)
```
I would prefer to have:
```go
func Coordinates() (long int, lat int)
```
rather than the unnamed version.